advanced/Scripts/piholeCheckout.sh

```diff
@@ -42,6 +42,11 @@ warning1() {
     esac
 }
 
+updateCheckFunc() {
+/opt/pihole/updatecheck.sh
```

Comment:
Indent the code inside the function please :)

Reply:
done